### PR TITLE
Feature South Africa

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

- [x] Add maps
- [x] Add Population data

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

screencapture-localhost-8000-profiles-country-za-south-africa-2018-10-16-15_11_55

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
It's missing from `/profiles` (Kenya and Ethiopia as well)

peek 2018-10-16 16-47

**Comment by kilemensi** [comment]
The country profile does not have an icon (Ethiopia as well)

screenshot from 2018-10-16 16-46-14

**Comment by kilemensi** [comment]
Census year is missing

screenshot from 2018-10-16 16-47-48

File: sa_sex.csv [hunk 1]
```diff
@@ -0,0 +1,21 @@
+level,geography,year,Population_Sex,total
```
Comment (kilemensi) [inline sa_sex.csv]: This file should be removed.

File: takwimu/sql/crime.sql [hunk 2]
```diff
@@ -0,0 +1,636 @@
...
+-- Data for Name: crime; Type: TABLE DATA; Schema: public; Owner: -
+--
+
+COPY crime (geo_level, geo_code, crime, total, geo_version) FROM stdin;
+province	KZN	Theft out of or from motor vehicle	18289	2009
+province	EC	Attempted murder	1858	2009
+province	MP	Assault with the intent to inflict grievous bodily harm	10803	2009
+province	GT	Driving under the influence of alcohol or drugs	26100	2009
+province	LIM	Common assault	9509	2009
+province	WC	Assault with the intent to inflict grievous bodily harm	24846	2009
```
Comment (kilemensi) [inline takwimu/sql/crime.sql]: Long names will not render correctly in viz

File: takwimu/sql/population_residence.sql [hunk 3]
```diff
@@ -224,30 +224,6 @@ level1	KE_1_027	2009	rural	548620
 level1	KE_1_038	2009	rural	380517
 level1	KE_1_008	2009	rural	565086
 level1	KE_1_024	2009	rural	469994
-country	ET	2009	urban	11956170
```
Comment (kilemensi) [inline takwimu/sql/population_residence.sql]: I've noticed you're removing `ET` data. Any reason?

Comment (phillipahereza) [reply takwimu/sql/population_residence.sql]: Resolved

**Pull request merged**